samples/text/text.cpp

```diff
@@ -283,6 +283,8 @@ class MyFrame: public wxFrame
 #endif // wxUSE_LOG
     void OnFileSave(wxCommandEvent& event);
     void OnFileLoad(wxCommandEvent& event);
+    void OnFileSaveRtf(wxCommandEvent& event);
```

Comment:
Why is in these two methods RTF spelled "Rtf", unlike everywhere else?

Reply:
Fixed, thanks.